manifests/init.pp

```diff
@@ -0,0 +1,44 @@
+# Top level class for ClickHouse DBMS installation and management.
+#
+# @summary this class allows you to install ClickHouse DB's repo, client and server
+#
+# @param server [Boolean]
+#   Is clickhouse-server should be installed or not. It won't be installed by default, because some time you need the client only.
+# @param client [Boolean]
+#   Is clickhouse-client should be installed or not.
+# @param repo [Boolean]
+#   Is apt or yum repository should be managed. Set to `false` by default to not affect your own repositories policy.
+#
+# @example Simple use
+#   include clickhouse
+#
+# @example Install server and client
+#   class { 'clickhouse':
+#     server => true,
+#   }
+#
+# @example Install everything and manage repository
+#   class { 'clickhouse':
+#     server      => true,
+#     server      => true,
```

Comment:
`server` is duplicated. maybe you meant `client`?

Reply:
exactly, thanks